packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -846,9 +846,19 @@ abstract class SliderComponentShape {
 ///  * [SliderTrackShape] Base component for creating other custom track
 ///    shapes.
 class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const RectangularSliderTrackShape();
+  /// Create a slider track that draws a circle 2 rectangles.
```

Comment:
Create a slider track that draws ~a circle~ 2 rectangles.

Reply:
done